src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.css

```diff
@@ -118,3 +118,7 @@ a, a:visited {
   white-space: nowrap;
   overflow: hidden;
 }
+
+.container {
+  font-size: 14px;
```

Comment:
What's this change for?

Reply:
It is the size of the expandable button which was bigger than the rest